proto/panacea/datadeal/v2alpha2/query.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package panacea.datadeal.v2alpha2;
+
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+import "panacea/datadeal/v2alpha2/deal.proto";
+import "panacea/datadeal/v2alpha2/datasale.proto";
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Deal returns a Deal.
+  rpc Deal(QueryDealRequest) returns (QueryDealResponse) {
+    option (google.api.http).get = "/panacea/datadeal/v2alpha2/deals/{deal_id}";
+  }
+  // Deals return a list of Deal.
+  rpc Deals(QueryDealsRequest) returns (QueryDealsResponse) {
+    option (google.api.http).get = "/panacea/datadeal/v2alpha2/deals";
+  }
+  // DataSale returns a DataSale.
+  rpc DataSale(QueryDataSaleRequest) returns (QueryDataSaleResponse) {
+    option (google.api.http).get = "/panacea/datadeal/v2alpha1/datasales/{deal_id}/{seller_cid}";
```

Comment:
```suggestion
    option (google.api.http).get = "/panacea/datadeal/v2alpha2/datasales/deals/{deal_id}/sellers/{seller_cid}";
```

typo and recommendation for rest api

Reply:
The comments were resolved in [5ff9743](https://github.com/medibloc/panacea-core/pull/360/commits/5ff97435c8e26791c520842c6cbcc305a503af89).